template-for-repository/proofs/Makefile.common

```diff
@@ -222,6 +222,8 @@ GOTO_INSTRUMENT ?= goto-instrument
 VIEWER ?= cbmc-viewer
 MAKE_SOURCE ?= make-source
 VIEWER2 ?= cbmc-viewer
+CMAKE ?= cmake
+ARPA ?= @echo "You must set ARPA in Makefile-project-defines to run arpa in this project"; false
```

Comment:
This definition forces the use of arpa.  (The evaluation to false will make the invocation of make fail.) I think this is a mistake.  A project should be able to decide on its own whether it will make use of arpa and the targets defined here.